src/general.h

```diff
@@ -0,0 +1,91 @@
+/* -----------------------------------------------------------------------------
+
+ Unified STO projection / AOM overlap code
+
+ Beta version: 1.0
+ 1-Mar-2019
+
+ Orestis George Ziogos, UCL
+ o.ziogos@ucl.ac.uk
+
+ For more information, examine the README file in the parent directory.
+
+-------------------------------------------------------------------------------- */
+#define cmax_length 1000
+#define sub_length 200
+#define AngToBohr 1.8897259886
+#define equal_diff 1e-6
```

Comment:
Only 
`#define equal_diff 1e-6
`
and
`#include<math.h>
`
are actually used by the embedded C functions.

Try dropping all other defines, includes, and the struct definitions.